### ISSUE TITLE
Use unity values when ALE files have missing values

### ISSUE THREAD

**Issue by peegee91** [shidarin/cdl_convert#49]
We often come accross ALE files that miss some of the CDL info.  Sometimes, it could be clips with empty fields in ASC_SOP or ASC_SAT, some other times it could be either column completely missing from the ALE.  Most of the time, we want these missing info to default to unity and convert anyway.  That would be useful!  You could also add a switch to turn this feature on or off.

**Comment by shidarin** [comment]
By "unity" you mean do nothing? So 1.0 for Sat, gain and gamma, 0.0 for offset?

We can do that. There's already a "strict" mode that we can tie into.

**Comment by peegee91** [comment]
Yes, that's what I mean.  By unity, I mean values that would leave the image unchanged.  Many thanks.
